### PR TITLE
Add a Drupal9 recipe

### LINKED ISSUE #1993: Support Drupal 9
Currently, Lando doesn't seem to support Drupal 9.

```
$ lando init
? From where should we get your app's codebase? current working directory
? What recipe do you want to use?
backdrop
drupal6
drupal7
❯ drupal8
joomla
lamp
laravel
```

By default, we'd need support for PHP 7.3, MySQL 5.7 and Drush 10 to best support Drupal 9. See https://www.drupal.org/docs/9/how-drupal-9-is-made-and-what-is-included/environment-requirements-of-drupal-9

### PR DESCRIPTION
**Pull request by AaronFeledy**
Resolves #1993 
- [x] My code includes the latest code from the `master` branch
- [x] My code includes an update to the [CHANGELOG](https://github.com/lando/lando/tree/master/docs/help)
- [x] My code includes a [functional test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [ ] My code includes a [unit test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [x] My code includes documentation updates if applicable.
- [ ] My code passes relevant CI status checks
- [ ] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

### CODE REVIEW

File: docs/help/2020-changelog.md [hunk 1]
```diff
@@ -10,3 +10,4 @@
 **ALSO, STILL, SERIOUSLY, READ THE DOCS!: https://docs.lando.dev/**
 
 * Upgraded to Docker Desktop 2.2.0.0
```
Comment (uberhacker) [inline docs/help/2020-changelog.md]: Wasn't this dropped?

Comment (uberhacker) [reply docs/help/2020-changelog.md]: Never mind.  I guess it wasn't.

**Comment by labboy0276** [comment]
@pirog this looks good to me, also as a FYI here are the reqs for D9: https://www.drupal.org/docs/9/how-drupal-9-is-made-and-what-is-included/environment-requirements-of-drupal-9

**Comment by AaronFeledy** [comment]
I just updated the download links to the latest beta. After D9 launches, those can be updated to use the `latest` download link to always download the latest D9.

**Comment by smutlord** [comment]
alright this is getting close, ive merged into the consolidation branch and made a few tweaks
https://github.com/lando/lando/pull/2251/files

**Pull request closed without merging**